web_readonly_bypass/static/src/js/readonly_bypass.js

```diff
@@ -0,0 +1,140 @@
+(function(){
+    var instance = openerp;
```

Comment:
Can you declare your JS as js-strict? https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#javascript